newIDE/app/src/UI/Theme/index.js

```diff
@@ -1,23 +1,74 @@
 // @flow
+import { createMuiTheme } from '@material-ui/core/styles';
+import { isLtr } from '../../Utils/i18n/RtlLanguages';
+import DefaultTheme, {
+  themeName as defaultThemeName,
+  type Theme,
+} from './DefaultTheme';
 import DarkTheme from './DarkTheme';
-import DefaultTheme from './DefaultTheme';
 import NordTheme from './NordTheme';
 import SolarizedDarkTheme from './SolarizedDarkTheme';
-import { type Theme } from './DefaultTheme';
 import './Global.css';
 
 // To add a new theme:
 // * copy the folder of an existing one (DarkTheme for example),
 // * import it at the top of the file
 // * add it below:
 export const themes = {
-  'GDevelop default': DefaultTheme,
+  [defaultThemeName]: DefaultTheme,
   Dark: DarkTheme,
   Nord: NordTheme,
   'Solarized Dark': SolarizedDarkTheme,
 };
 
-export const getTheme = (themeName: string): Theme =>
-  themes[themeName] || themes['GDevelop default'];
+export function getTheme({ themeName, language }: {| themeName: string, language: string |}): Theme {
+  let theme = themes[themeName];
+
+  if (!theme) {
+    console.warn(`Theme '${themeName}' is unavailable; '${defaultThemeName}' is used`);
+    theme = themes[defaultThemeName];
+  }
+
+  const ltr = isLtr(language);
+  const { gdevelopTheme, muiTheme } = theme;
+  return {
+    gdevelopTheme,
+    muiTheme: ltr ? createLtrTheme(muiTheme) : createRtlTheme(muiTheme),
+  };
+};
 
 export type GDevelopTheme = $PropertyType<Theme, 'gdevelopTheme'>;
+
+function createLtrTheme(muiTheme: object): object {
```

Comment:
Use `Object` in flow, `object` is considered as an error:
![image](https://user-images.githubusercontent.com/1280130/94209318-e7d81f80-fec3-11ea-9812-a9e4b77f0fb4.png)